controllers/keystoneapi_controller.go

```diff
@@ -297,6 +300,18 @@ func (r *KeystoneAPIReconciler) SetupWithManager(ctx context.Context, mgr ctrl.M
 		return err
 	}
 
+	// index networkAttachmentsField
+	if err := mgr.GetFieldIndexer().IndexField(context.Background(), &keystonev1.KeystoneAPI{}, networkAttachmentsField, func(rawObj client.Object) []string {
+		// Extract the secret name from the spec, if one is provided
```

Comment:
It isn't a `Secret` in the case, right?  Maybe we should change the comment.

Reply:
yes, updated